zeppelin-interpreter/src/main/java/org/apache/zeppelin/helium/HeliumPackage.java

```diff
@@ -18,13 +18,16 @@
 
 import org.apache.zeppelin.annotation.Experimental;
 
+import java.util.Map;
+
 /**
  * Helium package definition
  */
 @Experimental
 public class HeliumPackage {
   private HeliumType type;
   private String name;           // user friendly name of this application
+  private String version;
```

Comment:
In similar context, 

> It would be great if helium package has version field considering local package and latest
> (from https://github.com/apache/zeppelin/pull/1935#issuecomment-276862781)